apps/functions-subscription/src/main.ts

```diff
@@ -87,3 +100,13 @@ if (config.subscriptionRequest.consumer === 'on')
     cardinality: 'many',
     handler: makeSubscriptionRequestConsumerHandler(env),
   });
+
+if (config.events.producer === 'on')
+  app.cosmosDB('eventProducer', {
+    connection: 'EventProducerCosmosConnection',
+    databaseName: config.cosmosdb.databaseName,
+    containerName: config.cosmosdb.containersNames.subscriptionHistory,
+    leaseContainerName: config.cosmosdb.containersNames.leases,
+    leaseContainerPrefix: 'eventProducer',
```

Comment:
**note**: if you want to better separate the id of the lease document, [check this comment out](https://github.com/pagopa/trial-system/pull/52#discussion_r1633011919)

Reply:
Done ✅! Thanks!